libs/api/domains/driving-license/src/lib/graphql/main.resolver.ts

```diff
@@ -103,11 +107,11 @@ export class MainResolver {
   }
 
   @Query(() => ApplicationEligibility)
-  drivingLicenseApplicationEligibility(
+  async drivingLicenseApplicationEligibility(
```

Comment:
You don't need the async/await in this case, when the Promise function is directly returned